tests/unit/mixins/route-test.js

```diff
@@ -87,13 +85,12 @@ test('it can not use infinityModel without a Model Name', assert => {
       query() {}
     }
   });
-
-  assert.throws(() => {
+ 
+  try {
```

Comment:
assert.throws failes after ember 2.11 so wrap try catch.  However, this method does not work in the infinity-loader unit test.